### PR TITLE
Prepare 0.9 release

### PR DESCRIPTION
**Pull request by chriseclectic**
### Summary

Organize release notes for 0.9 release

### CODE REVIEW

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+prelude: |
+    The 0.9 release includes new backend options for parallel exeuction
+    of large numbers of circuits on a HPC cluster using a Dask distributed,
+    along with other general performance improvements and bug fixes.
+features:
+  - |
+    Add qiskit library SXdgGate and CUGate to the supported basis gates for
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:8]:
```suggestion
    Add qiskit library :class:`~qiskit.circuit.library.SXdgGate`
    and :class:`~qiskit.circuit.library.CUGate` to the supported basis gates for
```

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    the Aer simulator backends. Note that the CUGate gate is only natively
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:9]:
```suggestion
    the Aer simulator backends. Note that the :class:`~qiskit.circuit.library.CUGate`
    gate is only natively
```

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    supported for the statevector and unitary methods. For other simulation
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:10]:
```suggestion
    supported for the ``statevector`` and ``unitary`` methods. For other simulation
```

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    :class:`qiskit.circuit.library.generalized_gates.PauliGate`) to all
+    simulation methods of the
+    :class:`~qiskit.providers.aer.AerSimulator` and
+    :class:`~qiskit.providers.aer.QasmSimulator`.
+deprecations:
+  - |
+    Passing an assembled qobj directly to the
+    :meth:`~qiskit.providers.aer.AerSimulator.run` method of the Aer simulator
+    backends has been deprecated in favor of passing transpiled circuits
+    directly as `backend.run(circuits, **run_options)`.
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:23]:
```suggestion
    directly as ``backend.run(circuits, **run_options)``.
```

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  - |
+    All snapshot instructions in :mod:`qiskit.providers.aer.extensions` have
+    been deprecated. For replacement use the save instructions from the
+    :mod:`qiskit.providers.aer.library` module.
+  - |
+    Adding non-local quantum errors to a
+    :class:`~qiskit.providers.aer.noise.NoiseModel` has been deprecated due to
+    inconsistencies in how this noise is applied to the optimized circuit.
+    Non-local noise should be manually added to a scheduled circuit in Qiskit
+    using a custom transpiler pass before being run on the simulator.
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:33]: An example here would be good.

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+    ``["id", "rz", "sx", "cx"]`` due to the deprecation of the ``u3`` circuit
+    method in qiskit-terra and change of qiskit-ibmq-provider backend basis
+    gates. To use the old basis gates you can initialize a noise model with
+    custom basis gates as ``NoiseModel(basis_gates=["id", "u3", "cx"])``.
+  - |
+    Removed the ``backend_options`` kwarg from the ``run`` methnod of Aer backends
+    that was deprecated in qiskit-aer 0.7. All run options must now be passed as
+    separate kwargs.
+  - |
+    Removed passing `system_model` as a positional arg for the ``run`` method of the
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:53]:
```suggestion
    Removed passing ``system_model`` as a positional arg for the ``run`` method of the
```

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+    option value for ``max_parallel_experiments=1``. Previously this was treated
+    the same as ``max_parallel_experiments=0``.
+  - |
+    Fixes bug in the ``extended_stabilizer`` simulation method where it
+    incorrectly treated qelay gate and multi-qubit Pauli instructions as
+    unsupported.
+  - |
+    Fixes typo in the :class:`~qiskit.providers.aer.AerSimulator` and
+    :class:`~qiskit.providers.aer.QasmSimulator` options for the
+    ``extended_stabilizer_norm_estimation_repetitions`` option.
```
Comment (mtreinish) [inline releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:75]: Are there any bug numbers you want to link back to for these?

File: releasenotes/notes/0.9/custom-executor-3d0048b4dd4fa722.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+       backend = AerSimulator(max_job_size=1, executor=custom_executor)
+       job = backend.run(circuits)
+    
+    will split the exection into multiple jobs each containing a single
+    circuit. If job splitting is enabled the ``run`` method will return a
+    :class:`~qiskit.providers.aer.jobs.AerJobSet` object containing all the
+    individual :class:`~qiskit.providers.aer.jobs.AerJob` classes. After all
+    individual jobs finish running the job results are automatically combined
+    into a single Result object that is returned by `job.result()`.
```
Comment (mtreinish) [inline releasenotes/notes/0.9/custom-executor-3d0048b4dd4fa722.yaml:21]:
```suggestion
    into a single Result object that is returned by ``job.result()``.
```

or

```suggestion
    into a single Result object that is returned by
    :meth:`~qiskit.providers.jobs.Aer.Job.result()`.
```

File: releasenotes/notes/0.9/custom-executor-3d0048b4dd4fa722.yaml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+    Supported executors include Python multiprocessing executors
+    (eg. ``ThreadPoolExecutor``, ``ProcessPoolExecutor``), and
```
Comment (mtreinish) [inline releasenotes/notes/0.9/custom-executor-3d0048b4dd4fa722.yaml:24]: Maybe link to the python docs for these?

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml (hunk 5 above)
Comment (chriseclectic) [reply releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:33]: I don't have one, and it likely would be too complicated for a release note anyway.

File: releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml (hunk 7 above)
Comment (chriseclectic) [reply releasenotes/notes/0.9/0.9-release-0ecba93a5191e7ec.yaml:75]: Don't think so, they weren't in original release note files.

File: releasenotes/notes/0.9/mps-e81837c4acd0e0f0.yaml [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+    simulation method which did not correctly support permutations in the ordering of
+    the qubits on which the gate is applied.
+  - |
+    Fixes an issue where gate fusion could still be enabled for the
+    ``matrix_product_state`` simulation method even though it is not supported. 
+    Now fusion is always disabled for this method.
+  - |
+    Fixed bug in the ``matrix_product_state`` simulation method in computing the
+    normalization following truncation of the  Schmidt coefficients after
+    performing the SVD.
```
Comment (mtreinish) [inline releasenotes/notes/0.9/mps-e81837c4acd0e0f0.yaml:22]: Same thing about bug numbers from above

**Review by mtreinish — APPROVED** [review]
LGTM now, although I missed the mps file in my earlier review

**Pull request merged**